bindings/CMakeLists.txt

```
@@ -17,7 +17,7 @@ if(pyroot)
   endif()
 endif()
 
-if(tpython)
+if(tpython AND NOT (DEFINED ROOT_WHEEL_BUILD AND ROOT_WHEEL_BUILD))
```

```suggestion
if(tpython)
```
Why not just do `tpython=OFF` in the setup.py?